src/profile-logic/profile-data.js

```diff
@@ -154,19 +164,28 @@ export function getCallNodeInfo(
           funcIndex,
           categoryIndex,
           subcategoryIndex,
-          windowID
+          windowID,
+          inlinedIntoSymbol
         );
         prefixCallNodeAndFuncToCallNodeMap.set(
           prefixCallNodeAndFuncIndex,
           callNodeIndex
         );
-      } else if (category[callNodeIndex] !== categoryIndex) {
-        // Conflicting origin stack categories -> default category + subcategory.
-        category[callNodeIndex] = defaultCategory;
-        subcategory[callNodeIndex] = 0;
-      } else if (subcategory[callNodeIndex] !== subcategoryIndex) {
-        // Conflicting origin stack subcategories -> "Other" subcategory.
-        subcategory[callNodeIndex] = 0;
+      } else {
+        if (
+          sourceFramesInlinedIntoSymbol[callNodeIndex] !== inlinedIntoSymbol
+        ) {
+          // Conflicting inlining: -1.
+          sourceFramesInlinedIntoSymbol[callNodeIndex] = -1;
```

Comment:
Hm, in which case can this happen?
Edit: Just learned the answer from the comment on the type. Maybe it could be good to add a comment here as well?

Reply:
Done